### ISSUE TITLE
Not getting order total amount in order list api response

### ISSUE THREAD

**Issue by onamkrverma** [medusajs/medusa#4246]
### Describe the bug
Customer's [order list](https://docs.medusajs.com/api/store#tag/Customers/operation/GetCustomersCustomerOrders) api does not give order total amount, subtotal and other total amount in api response.

### System information

Medusa version (including plugins): ^1.10.0
Node.js version: v18.16.0
Database: 
Operating system: 
Browser (if relevant):

### Steps to reproduce the behavior
1. Login customer account with credential
2. Fetch order list of customer 'http://localhost:9000/store/customers/me/orders?expand=items'

### Expected behavior

According to medusa docs [order list](https://docs.medusajs.com/api/store#tag/Customers/operation/GetCustomersCustomerOrders) api will give `total` order amount, `subtotal`, `shipping_total` and other total like `tax_total`, `refunded_total` but i am not getting these total in api response.
 
### Screenshots

**Getting below response after fetching [order list](https://docs.medusajs.com/api/store#tag/Customers/operation/GetCustomersCustomerOrders) api**  

```
orders: [
{
      "object": "order",
      "id": "order_id",
      "shipping_address": {...},
      "fulfillments": [],
      "items": [
        {...}
      ],
      "shipping_methods": [],
      "discounts": [],
      "customer": {...},
      "payments": [
       {...}
      ],
      "region": {...},
      "swaps": [],
      "claims": [],
      "gift_cards": [],
      "gift_card_transactions": [],
      "refunds": [],
      "created_at": "2023-05-30T13:33:23.096Z",
      "status": "pending",
      "fulfillment_status": "not_fulfilled",
      "payment_status": "captured",
      "display_id": 93,
      "cart_id": "cart_id",
      "customer_id": "cus_id",
      "email": "email",
      "region_id": "reg_id",
      "currency_code": "inr",
      "tax_rate": null
    }
] 

```

**Comment by tsvetann** [comment]
I can confirm this issue. Asked on discord without any luck

**Comment by adrien2p** [comment]
should be fixed by [this](https://github.com/medusajs/medusa/pull/4294) just waiting approval

**Comment by tsvetann** [comment]
> should be fixed by [this](https://github.com/medusajs/medusa/pull/4294) just waiting approval

Thanks for fixing this issue. Just out of curiosity how did items.refundable broke the price calculation? Thx

**Comment by adrien2p** [comment]
This is because it was going through the legacy decorate totals if there is at least one total requested. The legacy only compute the requested totals, in this cas the items.refundable, but there were some perf issues which is why we moved to the new one which is more performant and allow to compute all the totals in less time.

**Comment by olivermrbl** [comment]
Solved in #4294
